src/webapp/helpers/time.js

```diff
@@ -1,8 +1,15 @@
-import { differenceInDays, fromUnixTime } from "date-fns";
+import { differenceInDays, parseISO } from "date-fns";
 
-const daysSinceSlackMessage = (slackTs) => {
```

Comment:
You might need to check if this function is used anywhere else in the codebase.

Also, consider writing some tests for this if you can, this will help a lot with code coverage!